ghe-app-manifest.template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ghe-app-for-jira",
+  "url": "https://github.com/marketplace/jira-software-github",
+  "redirect_url": "<<APP_HOST>>/github/manifest/<<UUID>>/complete",
+  "hook_attributes": {
+    "url": "<<APP_HOST>>/github/<<UUID>>/events"
+  },
+  "setup_url": "<<APP_HOST>>/github/<<UUID>>/setup",
```

Comment:
Can you explain the flow here?  UUID is setup beforehand but the app is not installed yet, so what's the point of having the UUID there?

Reply:
Also, keep in mind that we might use this app manifest for our own dev flow.  As might want to keep it a bit more abstracted?

Reply:
@mboudreau do you have thoughts on how we should handle the url value here? This is required and is meant to link to the apps homepage. However, we don't have the app name at this point (even if we prepopulate, people can edit it). The best I can see us doing is entering `<<APP_HOST>>/apps/` and adding messaging somewhere in the auto flow to let customers know they will need to update this in their settings manually?

Reply:
Maybe we should get together on this, but my understanding is that we shouldn't create any apps on our end until the github flow is done, which will provide us with everything we need to create the app in our DB.  Creating the app beforehand means that the process might not finish and we'll have apps that were never created...